### PR TITLE
win_disk_facts: Prefix facts with ansible_ and use raw values

### LINKED ISSUE #35325: win_disk_facts: facts are not being ansible_ namespaced
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
win_disk_facts

##### ANSIBLE VERSION
v2.5

##### SUMMARY
So we established that facts need to be namespaced using a `ansible_` prefix. If not they may be conflicting/overriding user variables. Since this module is exposing its information as `disks` that's a problem, it ought to be `ansible_disks`.

### LINKED ISSUE #35329: win_disk_facts: Should expose raw values
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
win_disk_facts

##### ANSIBLE VERSION
v2.5

##### SUMMARY
Currently there's no way to get the disk size in a format you can manipulate. For instance, the disk ought to be a 240GB disk. However the value is returned as:
```yaml
                "size": "223.57088470459GiB", 
```
But since it is a string, we cannot convert to a float and recalculate in GB. Which we could be doing like this:
```yaml
  - name: Isolate disk
    set_fact:
      disk: '{{ ansible_facts.disks|selectattr("system_disk")|first }}'

  - name: Set disk size
    set_fact:
      disk_size: '{{ (disk.size|float*1024*1024/1000/1000)|round|int }}GB'
```

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
This fixes #35325 
This fixes #35329 

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
win_disk_facts

##### ANSIBLE VERSION
v2.5

### CODE REVIEW

File: lib/ansible/modules/windows/win_disk_facts.ps1 [hunk 1]
```diff
@@ -39,7 +36,7 @@ try {
     Fail-Json -obj $result -message "Failed to search the disks on the target: $($_.Exception.Message)"
 }
 [int32]$diskcount = $disks | Measure-Object | Select-Object  -ExpandProperty Count
-$result.ansible_facts.total_disks = $diskcount
+#$result.ansible_facts.total_disks = $diskcount
```
Comment (jborean93) [inline lib/ansible/modules/windows/win_disk_facts.ps1]: May as well just remove the line if you are not going to return it anymore.

Comment (dagwieers) [reply lib/ansible/modules/windows/win_disk_facts.ps1]: That was the intention, thanks for catching this :-)

**Comment by marqelme** [comment]
@dagwieers 
Here is a mistake:
Fail-Json -message "Cmdlet was not started with elevated rights"

Right phrase has to be:
Fail-Json -message "Module was not started with elevated rights"

**Comment by dagwieers** [comment]
@marqelme It's not necessarily wrong, but indeed it's a more convenient message for the end-user.

**Comment by jhawkesworth** [comment]
I think this should be merged.
Would be nice if we had some size filters bytes_to_gib / bytes_to_gig kind of thing but the examples cover the conversions, so no actual need for special purpose filters.

**Review by jborean93 — APPROVED** [review]

**Pull request merged**